calicoctl/commands/apply.go

```diff
@@ -43,6 +43,9 @@ Options:
   -c --config=<CONFIG>      Path to the file containing connection
                             configuration in YAML or JSON format.
                             [default: ` + constants.DefaultConfigPath + `]
+  -n --namespace=<NS>       Namespace of the resource.
+                            Only applicable to NetworkPolicy and WorkloadEndpoint.
+                            [default: default]
```

Comment:
Once you take the fixed libcalico-go you should remove the `[default: default]` tag and just state the it will use the default namespace.  As discussed, use the CLI-configured namespace as the one that you use to create the resource-specific client.